.github/workflows/main.yml

```diff
@@ -25,3 +25,36 @@ jobs:
       - uses: actions/checkout@v1
       - uses: './'
         env: ${{matrix.env}}
+  industrial_ci_external:
+    strategy:
+      matrix:
+        env:
+          - {ROS_DISTRO: kinetic, _EXTERNAL_REPO: 'github:ros-industrial/industrial_core@kinetic-devel'}
+          - {ROS_DISTRO: kinetic, _EXTERNAL_REPO: 'gh:ros-industrial/motoman_experimental#kinetic-devel', UPSTREAM_WORKSPACE: '.travis.rosinstall -ros-industrial/industrial_experimental/IRC_v2', ROS_REPO: ros}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - shell: bash
+        env: ${{matrix.env}}
+        run: |
+          ./industrial_ci/_wrap_test.sh
```

Comment:
Instead of _wrap_test.sh, we could checkout the target directly in a checkout action

Reply:
Done in 580ff0b24dd1b475467e7e4b84d604e058945239